dataland-frontend/src/components/messages/SuccessUpload.vue

```diff
@@ -14,7 +14,10 @@ export default {
   name: "SuccessUpload",
   components: { Message },
   props: {
-    message: String,
+    message: {
+      type: String,
+      default: "",
```

Comment:
I am not 100 % sure but maybe you could delete the default value here, because it seems ok for me that "message" is undefined as long as there is no message passed to this component.

And you probably should also be able to delete the "v-if="message"" in line 5. Because if "message" is undefined, there will be no text anyway, right?

Correct me if I am wrong. Just some ideas.

Reply:
This is in case the date is null which shouldn't happen but if it does it won't throw an error

Reply:
Yes, you are right, there will be no "can't read null" error, just a blank tag will be displayed <h4></h4>